app/hooks/useLiquidityPosition.ts

```diff
@@ -392,14 +333,14 @@ export function useLiquidityPosition({
           )
             .amount()
             .toNumber();
-          return await cosmosTransfer(inbound.address, cosmosAmount, memo);
+          return await cosmosTransfer(inbound!.address, cosmosAmount, memo);
```

Comment:
Maybe we should check if inbound is undefined and throw an error on that situation, try to send to undefined address could be risk depends on wallet implementation